java/kafka/consumer/src/main/java/KafkaConsumerConfig.java

```diff
@@ -17,6 +19,18 @@ public class KafkaConsumerConfig {
     private static final Logger log = LogManager.getLogger(KafkaConsumerConfig.class);
 
     private static final long DEFAULT_MESSAGES_COUNT = 10;
+    private static final String DEFAULT_SERVICE_NAME = "my-cluster-kafka";
+    private static final String DEFAULT_EXPORTER_NAMES = "jaeger";
+    private static final String DEFAULT_METRICS_NAMES = "none";
+
+    private static final boolean DEFAULT_OPENTRACING_ENABLED = true;
+    private static final boolean DEFAULT_OPENTELEMETRY_ENABLED = true;
```

Comment:
You cannot enable the usage of OpenTracing and OpenTelemetry at the same time.
Other than this, the tracing should be disabled by default. It's not something that the users usually have.

Related to this, I would use an approach like the bridge: we have a `bridge.tracing` property where you can specify `jaeger` or `opentelemetry` or not specifying anything if you don't want tracing.
Not saying we need an application property but we can use an env var, something like `TRACING_SYSTEM` or like that. It could get only `jaeger` or `opentelemetry` as values, or not being defined for disabling tracing.

Reply:
TRACING_SYSTEM introduced. Other envVars commented out